app/assets/v2/js/grants/fund.js

```diff
@@ -23,7 +30,7 @@ $(document).ready(function() {
 
   updateSummary();
 
-  $('.nav-item').click(function(e) {
+  $('#grants_form .nav-item').click(function(e) {
```

Comment:
this kind of rules will be a problem when we start creating a bundle js since we are not going to control this being loaded in other areas, use custom classes or data attr

Reply:
agreed this would need to be fixed in a follow up PR to use what the tabs we already have
It's creating random weird bugs in css cause of that 

cc @owocki